libs/remix-ui/solidity-compiler/src/lib/solidity-compiler.tsx

```diff
@@ -43,7 +43,7 @@ export const SolidityCompiler = (props: SolidityCompilerProps) => {
       const hide = await api.getAppParameter('hideWarnings') as boolean || false
       setHideWarnings(hide)
     })()
-  }, [])
+  })
```

Comment:
@Aniket-Engg By removing the dependency array, this useEffect will be fired every single time this component is re-rendered. I don't think it is a good solution in my opinion, we can tie the change to a variable and pass it to the dependency array. `, [hideWarnings])`

Reply:
I tried that. It is not solving the issue